.readthedocs.yml

```diff
@@ -3,8 +3,13 @@ version: 2
 build:
   image: latest
 
+sphinx:
+  builder: html
+  configuration: docs/conf.py
+  fail_on_warning: true
+
 python:
-  version: 3.7
+  version: 3.9
```

Comment:
I remember trying Python 3.9 a while back but it failed for some reason and I had to fall back to 3.8. If it works now, great!

Reply:
🤞

Reply:
Falling back to 3.8